packages/components/src/Input/InputSearch/InputSearch.module.scss

```diff
@@ -44,6 +38,12 @@ $classname--input: '.input[type="search"]';
   font-size: 1rem;
   line-height: 1.5;
 
+  @include form-input-reset;
```

Comment:
Will the reset coming after line 38 and 39 mean the font-size and line-height are superseded by the mixin?

Reply:
Good catch! I'll move this one back. Not sure why I moved it to begin with 😬